packages/docusaurus-plugin-content-blog/src/pluginOptionSchema.ts

```diff
@@ -76,12 +76,13 @@ export const PluginOptionSchema = Joi.object({
     DEFAULT_OPTIONS.beforeDefaultRehypePlugins,
   ),
   feedOptions: Joi.object({
-    type: Joi.alternatives().conditional(
-      Joi.string().equal('all', 'rss', 'atom'),
-      {
+    type: Joi.alternatives()
+      .conditional(Joi.string().equal('all', 'rss', 'atom'), {
         then: Joi.custom((val) => (val === 'all' ? ['rss', 'atom'] : [val])),
-      },
-    ),
+      })
+      .empty(undefined)
+      .empty(null)
+      .default(['rss', 'atom']),
```

Comment:
Should use `DEFAULT_OPTIONS.feedOptions.type`, similarly to how we handle default options in other places

Reply:
Hi,

I've tried setting `DEFAULT_OPTIONS.feedOptions.type` to `'all'` as well as `['rss', 'atom']` and both give me these errors
![Screen Shot 2020-11-30 at 9 19 12 AM](https://user-images.githubusercontent.com/43725481/100621358-91d18f00-32ed-11eb-9e43-55fe0ebaa199.png)
![Screen Shot 2020-11-30 at 9 23 20 AM](https://user-images.githubusercontent.com/43725481/100621496-bcbbe300-32ed-11eb-81e9-aee30635fc5b.png)

Also, setting `DEFAULT_OPTIONS.feedOptions.type` to `['rss', 'atom']` also gives me an additional error `[ValidationError: "feedOptions.type" does not match any of the allowed types]`.

Do you have any suggestions as to how to resolve these? The only way I've found to make it work is by adding the .empty() method within the Joi object.

Thanks!

Reply:
I can't debug remotely code that I can't fully see, You'd rather commit your failed attempt and ask a new review.

> Also, setting DEFAULT_OPTIONS.feedOptions.type to ['rss', 'atom'] also gives me an additional error [ValidationError: "feedOptions.type" does not match any of the allowed types].

You should make the validator accept an array for this to work, currently it only accept strings

Reply:
Ok yeah, I'll make another commit and ask for a new review with the failed attempt.